tests/test_time_series_checker.py

```diff
@@ -78,7 +78,7 @@ def _run_test(
         value_column="value",
     )
     with engine.connect() as conn:
-        write_database(df, conn, schema.name, schema.time_config)
+        write_database(df, conn, schema.name, schema.time_config, if_table_exists="replace")
```

Comment:
Do you see the need for replace now?

Reply:
I think you did persuade me on its value, at least for tests. I had to add it here because of the changes to `write_database`. Previously, we submitted the write through Polars. It would create a new table automatically, if even the value of `if_table_exists` was append. That doesn't work now. We have to run a create table command. It only applies to tests. In the product code we create the table through sqlalchemy.

Reply:
That and if the user decides to use chronic interactively.

Reply:
I vote that we try to keep this function hidden from users. It could have disastrous results. All writes should go through the ingest functions so that we can run the checkers and ensure consistency.